File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    # get input from user between 1 to 8
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

  # print the digram
for i in range(n):
    print(" " * (n - 1 - i), end="")
    print("#" * (i + 1), end="")
    print(" " * 2, end="")
    print("#" * (i + 1), end="")
    print(end="\n")

File: pset1/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    int n;

    do
    {
        //user input
        n = get_int("height:");
        
    } 
    while( n<1 || n>8);  //for input accepint only between 1 to 8
    
    for(int i= 0; i<n ; i++)
    {
        for(int j=0;j<n; j++)
        {
            
            if(i + j<n-1) //for blanck space
            
              printf(" ");
              
            else
                printf("#");
        }
          
        printf("\n");    //for next line
    }

}